app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= link_to t(".the_form_contains") %>
+      <%= pluralize(@user.errors.size, "error") %>.
```

Comment:
Nghiên cứu I18n để view số nhiều số ít chỗ này nào em
https://viblo.asia/p/i18n-in-rails-part-2-GrLZDAO3lk0